src/sonic-yang-models/tests/yang_model_tests/yangModelTesting.py

```diff
@@ -201,12 +215,34 @@ def logStartTest(self, desc):
 
     """
         Load Config Data and return Exception as String
+
+        Parameters:
+            jInput (dict): input config to load.
+            verify (dict): contains xpath, key and value. This is used to verify,
+            that node tree at xpath contains correct key and value.
+            Example:
+            'verify': {'xpath': "/sonic-acl:sonic-acl/ACL_TABLE/ACL_TABLE_LIST\
+                        [ACL_TABLE_NAME='NO-NSW-PACL-V4']/stage",
+                        'key': 'sonic-acl:stage',
+                        'value': 'INGRESS'
+            }
     """
-    def loadConfigData(self, jInput):
+    def loadConfigData(self, jInput, verify=None):
         s = ""
         try:
             node = self.ctx.parse_data_mem(jInput, ly.LYD_JSON, \
             ly.LYD_OPT_CONFIG | ly.LYD_OPT_STRICT)
+            # verify the data tree if asked
+            if verify is not None:
+                xpath = verify['xpath']
+                set = node.find_path(xpath)
+                for dnode in set.data():
+                    if (xpath == dnode.path()):
+                        data = dnode.print_mem(ly.LYD_JSON, ly.LYP_WITHSIBLINGS \
+                            | ly.LYP_FORMAT | ly.LYP_WD_ALL)
+                        data = json.loads(data)
+                        assert (data[verify['key']] == verify['value'])
+                        s = 'verified'
```

Comment:
Why not just return true/false from this method to indicate loading was successful or not?

Reply:
All tests depend on Error string returned from libyang while loading. 80% tests are negative for yang, where loading fails, because of some must\pattern\when check failure. So for the tests where data is loaded successfully, I use None or verified[if verification is needed] string.